ni_measurementlink_service/_internal/grpc_servicer.py

```diff
@@ -83,7 +83,13 @@ def time_remaining(self) -> float:
 
     def abort(self, code: grpc.StatusCode, details: str) -> None:
         """Aborts the RPC."""
-        self._grpc_context.abort(code, details)
+        try:
+            self._grpc_context.abort(code, details)
+        except Exception:
```

Comment:
Handled it using `if isinstance(e, Exception):`

Reply:
- The comparison `if e == Exception():` fails because `Exception()` and `e` represent two distinct objects and are not considered equal.
- As a workaround, it's handled using `if type(e) is Exception:` to specifically check if the type of `e` is an instance of the `Exception` class.